### PR TITLE
373 change default genesis config of stellar relay pallet

### LINKED ISSUE #373: Change default genesis config of stellar-relay pallet
Changes the default configuration of the Tier 1 validator set as required in https://github.com/pendulum-chain/tasks/issues/94.

### PR DESCRIPTION
**Pull request by ebma**
Updates the default genesis config of the stellar relay pallet to match the latest configuration.

Closes #373.

### CODE REVIEW

File: pallets/stellar-relay/src/lib.rs [hunk 1]
```diff
@@ -174,8 +174,10 @@ pub mod pallet {
 			};
 			let organization_satoshipay =
 				OrganizationOf::<T> { name: create_bounded_vec("SatoshiPay"), id: 1.into() };
-			let organization_wirex =
-				OrganizationOf::<T> { name: create_bounded_vec("Wirex"), id: 2.into() };
+			let organization_franklin_templeton = OrganizationOf::<T> {
```
Comment (TorstenStueber) [inline pallets/stellar-relay/src/lib.rs:177]: Is there a problem that our current definition (the one used on Amplitude at the moment) uses a different order? There it is:

```
[
  {
    id: 0
    name: Stellar Development Foundation
  }
  {
    id: 1
    name: SatoshiPay
  }
  {
    id: 2
    name: Coinqvest
  }
  {
    id: 3
    name: Blockdaemon
  }
  {
    id: 4
    name: LOBSTR
  }
  {
    id: 5
    name: Public Node
  }
  {
    id: 6
    name: Franklin Templeton
  }
]
```

Maybe it's better to keep it consistent at least.

Comment (ebma) [reply pallets/stellar-relay/src/lib.rs]: Order shouldn't matter but I changed it anyways

**Review by TorstenStueber — APPROVED** [review]

File: pallets/stellar-relay/src/lib.rs (hunk 1 above)
Comment (TorstenStueber) [reply pallets/stellar-relay/src/lib.rs]: Thanks!

**Comment by ebma** [comment]
I'll merge this now even though the last CI job failed because it worked before the re-ordering and it's unlikely that the last commit broke something. Probably there just was some race condition in the tests.

**Pull request merged**